.gitignore

```diff
@@ -1,3 +1,3 @@
 .idea/*
 *.iml
-target/*
+target/*
```

Comment:
why?

Reply:
First I thought to add here yml files, but than I realized its unnecessary so I deleted it and accidentally delete with empty line